### PR TITLE
added flatMap

### PR DESCRIPTION
**Pull request by davidchase**

### CODE REVIEW

File: documentation/README.md [hunk 1]
```diff
@@ -30,6 +30,217 @@
...
+- [takeUntil](#takeuntil)
+- [takeWhile](#takewhile)
+- [test](#test)
+- [times](#times)
+- [uncurry](#uncurry)
+- [uncurry3](#uncurry3)
+- [values](#values)
+- [xor](#xor)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
```
Comment (stoeffel) [inline documentation/README.md]: whoops somehow the toc got duplicated.
can you clean that? 
1. remove the toc (not the doctoc comment!)
2. run `npm run toc`

**Comment by stoeffel** [comment]
Looks nice. This is good to go after the toc cleanup (see comment).

Thx a lot.

**Comment by davidchase** [comment]
let me know if i cleaned it correctly...

**Comment by stoeffel** [comment]
Yop.

Thx again :rainbow:

**Comment by davidchase** [comment]
sweet, glad to help great library :+1:

**Comment by stoeffel** [comment]
Btw. check messages on twitter😊

**Pull request merged**